free-programming-books-ar.md

```diff
@@ -32,39 +39,55 @@
 * [الإدارة المتقدمة لجنو/لينكس ](http://librebooks.org/gnu-linux-advanced-administration/)
 * [دفتر مدير دبيان](http://librebooks.org/debian-handbook-arabic/)
 * [دليل إدارة خواديم أوبنتو ](http://librebooks.org/ubuntu-server-guide-arabic/)
+* [كتاب لينكس الشامل](https://www.kutub.info/library/book/596)
+* [سطر اوامر لينكس](111000.net/systems/files/The_Linux_Command_Line-arabic-14.07.pdf) (PDF)
```

Comment:
sort order for lists is based on unicode- see issue #2503 where the issue is discussed.